src/elements/ProjectJobs/projectJobs.utils.js

```diff
@@ -14,7 +14,10 @@ export const getJobsStatistics = (jobs, match, scheduledJobs, workflows) => {
       0
     )
     jobsFailed = groupByUniqName(jobs.data, 'metadata.name').reduce(
-      (prev, curr) => (curr.status.state === 'error' ? (prev += 1) : prev),
+      (prev, curr) =>
+        curr.status.state === 'error' || curr.status.state === 'aborted'
+          ? (prev += 1)
```

Comment:
Do we really need the assignment here? 🤔 
This is a reduction function. It works with the return value.
Then also the parentheses are unneeded.
```suggestion
          ? prev + 1
```
Please see if this works.